airbyte-cdk/python/unit_tests/sources/test_source.py

```diff
@@ -7,10 +7,10 @@
 import tempfile
 from collections import defaultdict
 from contextlib import nullcontext as does_not_raise
-from typing import Any, List, Mapping, MutableMapping, Optional, Tuple
-from unittest.mock import MagicMock
```

Comment:
why is this import deleted?

Reply:
Using the `MagicMock()` already included in `mocker` instead of importing it was a [review suggestion](https://github.com/airbytehq/airbyte/pull/19977#discussion_r1044325754) in the previous PR